simulator/mips/t/unit_test.cpp

```diff
@@ -2680,6 +3147,44 @@ TEST_CASE( "MIPS32_instr: xori")
 }
 ////////////////////////////////////////////////////////////////////////////////
 
+TEST_CASE( "MIPS32_instr: xori 0 with 0")
```

Comment:
Please remove test case above, it is no longer useful.

Reply:
You mean remove test for `xori 0 with 0`?

Reply:
That one:

https://github.com/MIPT-ILab/mipt-mips/blob/472a8de763b4098977e7650d5414f41a1bd207e0/simulator/mips/t/unit_test.cpp#L3124-L3140

Reply:
Done